apps/graphql/src/types/output/Itinerary.js

```diff
@@ -77,10 +77,9 @@ const VehicleType = new GraphQLObjectType({
     uniqueNo: { type: GraphQLString },
   },
 });
-const ProviderType = new GraphQLObjectType({
-  name: 'ProviderType',
+const TransporterType = new GraphQLObjectType({
```

Comment:
Could we stick to one `GraphQLObjectType` pr file?

Reply:
👍 good point will be better for the future.